base/test_views.py

```diff
@@ -11,17 +14,17 @@ def tearDown(self):
         self.browser.quit()
 
     def test_home(self):
-        self.browser.visit('http://localhost:8000')
+        self.browser.visit(test_url)
         test_string = 'Hello, world!'
         if self.browser.is_text_present(test_string):
```

Comment:
This method, is not testing for the existance of the text properly, if you manually change it to a string you know is not part of the page, the test will pass.

The proper way to make this assertion is to do the following
`self.assertTrue(self.browser.is_text_present(test_string))` this actually breaks the test when looking for a non present string. Maybe creating another test, for checking the non presence of a string is a good idea for this very reason. 

This comment applies for all the cases in this file.